src/components/Dropdown/V2/Dropdown.types.ts

```diff
@@ -24,6 +24,7 @@ export interface DropdownState extends DropdownMenuDimensions {
 
 export interface DropdownProps extends DropdownMenuDimensions {
   activeClassName: string
+  allowMultipleSelection?: boolean
```

Comment:
@tinkertrain If it's a boolean, we don't need to make it optional. We'll just add a defaultProp of `false` :)

Reply:
Ah yeah... thanks!